Cargo.toml

```diff
@@ -54,4 +54,6 @@ members = [
     "who",
     "whoami",
     "yes",
+    "comm",
+    "join",
```

Comment:
Are this PR related to `join` as well?

Reply:
I started working on join and then made an initial commit, the changes were reflected here too.